tests/test_cli.py

```diff
@@ -26,13 +28,13 @@ def count_fastq_reads(path):
 
 
 def copy_config(source, target, genome_reference=None, read_mapper=None, duplicate_marker=None):
-    """Copy config, possibly changing genome_reference or read_mapper"""
+    """Copy config, possibly changing any non-None arguments"""
 
     with open(source) as infile:
         with open(target, "w") as outfile:
             for line in infile:
                 if genome_reference is not None and line.startswith("genome_reference:"):
-                    path = Path("testdata/chr1mini.fasta").absolute()
+                    path = Path(genome_reference).absolute()
                     line = f"genome_reference: {path}\n"
```

Comment:
You could remove the `path` variable and just use `genome_reference` directly in the f-string.  There is no need to make it into a absolute `Path` object, as this has already been done on assignment.

Reply:
I was also thinking it might be better to remove the `genome_reference` as input entirely and instead just convert the `test_config.yaml` default for `genome_reference` to it's absolute path. Then we can't change reference but we won't need to specify it in the script nor give it as input for all test functions which involve `run`. 

What do you think about that?

Reply:
Hm, I ran in some trouble with the different paths required from the pytest scripts and the bash script so I went with your suggestion